.github/workflows/main.yml

```diff
@@ -0,0 +1,100 @@
+name: ci
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+          - 3.8
+          - 3.6
+    
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: >-
+          sudo apt-get install -y
+          build-essential
+          php7.4-cli
+          haskell-platform
+          fp-compiler
+          python2.7
+          cppreference-doc-en-html
+          cgroup-lite
+          libcap-dev
+          python-dev
+          libpq-dev
+          libcups2-dev
+          libyaml-dev
+          rustc
+          mono-mcs
+
+      - name: Install requirements
+        run: |
+          pip install -r requirements.txt
+          pip install -r dev-requirements.txt
+
+          sudo $(which python) prerequisites.py -y --cmsuser=$USER install
+          python setup.py install
+
+      - name: Change group for working and temp directories to $USER
+        run: |
+          sudo chgrp $USER . /tmp
```

Comment:
By default, the user group is `docker` for GitHub actions runner. Without this, somehow CMS cannot create temp files in tests (permission denied)